_posts/2025-05-02-consultas-rut.md

```
@@ -0,0 +1,31 @@
+gh-repo: /capitalexpress/capitalexpress.github.io
+gh-badge: [star, fork, follow]
+tags: ["Reportes", "Cliente", "Ejecutivo", "Deudor", "Empresa", "Matriz de riesgo"]
+comments: false
+author: Catalina Pinilla
+---
+
+## Consulta por rut
+
+Ahora puedes consultar cualquier RUT, sin necesidad de que esté enrolado
+En Advance, hemos creado la sección `Consultas por RUT`, donde podrás obtener reportes de:
+- Cliente de `Capital Express Servicios Financieros
```

markdownlint-cli2 (0.17.2)

Lists should be surrounded by blank lines
null

(MD032, blanks-around-lists)